docs/adding_software/opening_pr.md

```diff
@@ -91,20 +101,19 @@ git push koala example_branch
 6) Go to the [GitHub web interface](https://github.com/EESSI/software-layer) to open your pull request,
    or use the helpful link that should show up in the output of the `git push` command.
 
-   **Make sure you target the correct branch**: the one that corresponds to the version of EESSI you want to add
-   software to (like `2023.06-software.eessi.io`).
+   **Make sure you target the `main` branch.**
 
    If all goes well, one or more bots :robot: should almost instantly create a comment in your pull request
```

Comment:
This is no longer true, since we've silenced all bots. So this should be removed, otherwise people may think something is wrong.